project-resources/project-directed-funding-policy.md

```diff
@@ -0,0 +1,7 @@
+# Project Directed Funding Policy
+
+All OpenJS Foundation projects must adhere to the Project Directed Funding policy as outlined on the OpenJS Foundation website:
+
+* [Project Directed Funding Policy](https://openjsf.org/projects/#project-directed-funding)
```

Comment:
this anchor doesn't exist

Reply:
Oops, yeah I had hoped to just add an anchor link on the website, but this is non trivial in prismic. Good catch.